examples/linear/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I think ^^^ package-lock.json's were going to be added to .gitignore and removed from the examples, btw.